### PR TITLE
[OpenShift] Update payloads for 1.5 release

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Add pipelines v0.24.3

Add triggers v0.14.2

Add Addons for version 1.5

Signed-off-by: Nikhil Thomas <nikthoma@redhat.com>

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: cmd/openshift/operator/kodata/tekton-trigger/0.14.2/02_tektoncd-triggers-interceptor-v0.14.2.yaml [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+---
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: ClusterInterceptor
+metadata:
+  name: gitlab
+spec:
+  clientConfig:
+    service:
+      name: tekton-triggers-core-interceptors
+      namespace: tekton-pipelines
```
Comment (savitaashture) [inline cmd/openshift/operator/kodata/tekton-trigger/0.14.2/02_tektoncd-triggers-interceptor-v0.14.2.yaml:57]: Does this namespace will be replaced to `openshift-pipelines` :thinking:

Comment (nikhil-thomas) [reply cmd/openshift/operator/kodata/tekton-trigger/0.14.2/02_tektoncd-triggers-interceptor-v0.14.2.yaml:57]: yes :+1:

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

**Pull request merged**